docs/java-development.rst

```diff
@@ -110,19 +110,33 @@ Release process
 Maintainer prerequisites
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-It is important to get familiar with the
-`OSSRH guide <https://central.sonatype.org/pages/ossrh-guide.html>`__ and
-especially the
-`Maven section for performing a release deployment <https://central.sonatype.org/pages/apache-maven.html>`__.
+To be able to maintain a Java component, a developer must have:
 
-To be able to maintain a Java component, a developer must:
+- a GitHub_ account with push rights to the GitHub source code repository
+- a Sonatype_ account and be registered as a maintainer of the
+  `org.openmicroscopy` repository
+- a valid PGP key for signing the tags and the JARs
+- a local :file:`~/.m2/settings.xml` file configured with an access token
+  generated as described in https://central.sonatype.org/publish/generate-token
 
-- have a GitHub_ account and have push rights to the GitHub source code
-  repository
-- have a Sonatype_ account and be registered as a maintainer of the
-  `org.openmicroscopy` repository (JIRA issues should be opened for each
-  developer)
-- have a valid PGP key for signing the tags and the JARs
+Follow the instructions at https://central.sonatype.org/register/legacy to
+create a Sonatype account allowing to publish via OSSRH. You need to
```

Comment:
Shall we include the code block?

Reply:
Withdrawing. @sbesson explained that the sonatype server provides this code block.